udpa/annotations/status.proto

```diff
@@ -10,6 +10,15 @@ extend google.protobuf.FileOptions {
   StatusAnnotation file_status = 222707719;
 }
 
+extend google.protobuf.MessageOptions {
```

Comment:
Instead of adding new annotations in the udpa directory, how about adding these in the xds directory, as per the agreed direction discussed in #2?

Reply:
Sure will do.